Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "prometheus-client"
-version = "0.22.2"
+version = "0.22.4"
```

```suggestion
version = "0.22.3"
```

Sorry, another pull request raced you to the version bump. Let's release this in `v0.22.3`.